### DISCUSSION TITLE
eventual ip leak when using http2 with proxies?

### DISCUSSION THREAD

**Discussion by sla-te** [encode/httpx#1738] [Potential Issue]
Is there a possibility, that when using http2 the real ip might leak? I am having some weird issues with a server, that I can only reach through a proxy...

**Comment by lovelydinosaur** [comment]
You've not really given enough information here for anyone to be able to help.

You'd need to describe what you're doing, and what behaviour you're seeing, etc...

**Comment by graingert** [comment]
AFAIK it's possible with http3 multipath QUIC
